openai/requirements.txt

```diff
@@ -0,0 +1 @@
+openai ~= 0.27
```

Comment:
The CI machinery should be working now and we've reached pack-related CI runs.

It's failing on `requirements.txt` now: https://github.com/StackStorm-Exchange/exchange-incubator/actions/runs/5879130202/job/15942547692?pr=179#step:8:1714

@dalarsen Maybe worth trying the requirements without whitespaces:

```suggestion
openai~=0.27
```
Otherwise, last resort is to go with the old notation:
```
openai>=0.27.0,<1.0
```

Reply:
Weird, it's still failing, but I can't reproduce the error locally at all?

`(venv) dalarsen@EXPWYFP79391M openai % pip install -r requirements.txt 
Collecting openai<1.0,>=0.27.0
  Using cached openai-0.27.8-py3-none-any.whl (73 kB)
Requirement already satisfied: requests>=2.20 in ./venv/lib/python3.9/site-packages (from openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (2.31.0)
Requirement already satisfied: aiohttp in ./venv/lib/python3.9/site-packages (from openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (3.8.5)
Requirement already satisfied: tqdm in ./venv/lib/python3.9/site-packages (from openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (4.66.1)
Requirement already satisfied: certifi>=2017.4.17 in ./venv/lib/python3.9/site-packages (from requests>=2.20->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (2023.7.22)
Requirement already satisfied: urllib3<3,>=1.21.1 in ./venv/lib/python3.9/site-packages (from requests>=2.20->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (2.0.4)
Requirement already satisfied: charset-normalizer<4,>=2 in ./venv/lib/python3.9/site-packages (from requests>=2.20->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (3.2.0)
Requirement already satisfied: idna<4,>=2.5 in ./venv/lib/python3.9/site-packages (from requests>=2.20->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (3.4)
Requirement already satisfied: attrs>=17.3.0 in ./venv/lib/python3.9/site-packages (from aiohttp->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (23.1.0)
Requirement already satisfied: multidict<7.0,>=4.5 in ./venv/lib/python3.9/site-packages (from aiohttp->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (6.0.4)
Requirement already satisfied: async-timeout<5.0,>=4.0.0a3 in ./venv/lib/python3.9/site-packages (from aiohttp->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (4.0.3)
Requirement already satisfied: frozenlist>=1.1.1 in ./venv/lib/python3.9/site-packages (from aiohttp->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (1.4.0)
Requirement already satisfied: aiosignal>=1.1.2 in ./venv/lib/python3.9/site-packages (from aiohttp->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (1.3.1)
Requirement already satisfied: yarl<2.0,>=1.0 in ./venv/lib/python3.9/site-packages (from aiohttp->openai<1.0,>=0.27.0->-r requirements.txt (line 1)) (1.9.2)
Installing collected packages: openai
Successfully installed openai-0.27.8
WARNING: You are using pip version 21.3.1; however, version 23.2.1 is available.
You should consider upgrading via the '/Users/dalarsen/PycharmProjects/public/exchange-incubator/openai/venv/bin/python -m pip install --upgrade pip' command.
(venv) dalarsen@EXPWYFP79391M openai % cat requirements.txt 
openai>=0.27.0,<1.0
`